Fix a bug in `io.fits` with fortran-ordered arrays in python3 (#8282)
astropy/astropy: CHANGES.rst

astropy.io.fits
^^^^^^^^^^^^^^^

> - Fix a bug in io.fits with writting fortan-ordered arrays in python3 when np.nditer was available [#8282]

Reviewer: Nitpicks to the change log text. :grimacing: Also, please move this entry to the section indicated in milestone.
```suggestion
- Fix a bug in ``io.fits`` with writting Fortan-ordered arrays in Python 3 when
  ``np.nditer`` was available. [#8282]
```

astropy.io.fits
^^^^^^^^^^^^^^^

> - Fix a bug in ``io.fits`` with writting Fortran-ordered arrays to file objects [#8282]

Reviewer 2: Typo: "writting" in case anyone wants to fix the changelog entry.

astropy.io.fits
^^^^^^^^^^^^^^^

> - Fix a bug in ``io.fits`` with writting Fortan-ordered arrays in Python 3

Reviewer: ```suggestion
- Fix a bug in ``io.fits`` with writting Fortran-ordered arrays in Python 3
```

Reviewer 3: Suggestion for the changelog: *Fix writing Fortran-ordered arrays with a file object* (`nditer` is in all supported versions of Numpy now i think, and the this also applies to Python 2 for v2.0.x)

Reviewer: @neutrinoceros , I think this is the remaining review comment to be addressed. And yes, @saimn has the final say in the approval process for this PR. Thank you very much!

Author: should be good now !

Reviewer 3: Almost good but please mention that this is only for writing to file objects, and I don't think that it is specific to Python 3 (the PR that changed the method with which file object are written was specific to Python 3, but this bug is not).

Author: my bad. Done :)

>   when np.nditer was available [#8282]

Reviewer: ```suggestion
  when ``np.nditer`` was available. [#8282]
```
One more nitpick... :sweat_smile:
